onnxruntime/core/providers/cpu/ml/tree_ensemble_aggregator.h

```diff
@@ -326,11 +326,11 @@ class TreeAggregatorMin : public TreeAggregator<InputType, ThresholdType, Output
 template <typename InputType, typename ThresholdType, typename OutputType>
 class TreeAggregatorMax : public TreeAggregator<InputType, ThresholdType, OutputType> {
  public:
-  TreeAggregatorMax<InputType, ThresholdType, OutputType>(size_t n_trees,
```

Comment:
All other similar class like TreeAggregatorMin and TreeAggregatorSum does not include type qualifier.